test/plots/piePlotTests.ts

```diff
@@ -2,92 +2,140 @@
+
+        let firstPathPoints1 = pathPoints1[0].split(",");
+        assert.operator(parseFloat(firstPathPoints1[0]), ">", 0, "draws line to the right");
+        assert.closeTo(parseFloat(firstPathPoints1[1]), 0, 1, "draws line horizontally");
+
+        let arcDestPoint1 = pathPoints1[1].split(",").slice(5);
+        assert.closeTo(parseFloat(arcDestPoint1[0]), 0, 1, "ends at x origin");
+        assert.operator(parseFloat(arcDestPoint1[1]), "<", 0, "ends above 0");
 
-    it("updates slices when data changes", () => {
-      let svg = TestMethods.generateSVG(500, 500);
-      let piePlot = new Plottable.Plots.Pie();
-      piePlot.sectorValue((d) => d.value);
-      let fourSliceData = [
-        { value: 1 },
-        { value: 1 },
-        { value: 1 },
-        { value: 1 }
-      ];
-      let dataset = new Plottable.Dataset(fourSliceData);
-      piePlot.addDataset(dataset);
-      piePlot.renderTo(svg);
-      let fourSlicePathStrings: String[] = [];
-      piePlot.content().selectAll("path").each(function() { fourSlicePathStrings.push(d3.select(this).attr("d")); });
-      assert.lengthOf(fourSlicePathStrings, fourSliceData.length * 2, "2 paths per datum");
-
-      let twoSliceData = [
-        { value: 1 },
-        { value: 1 }
-      ];
-      dataset.data(twoSliceData);
-      let twoSlicePathStrings: String[] = [];
-      piePlot.content().selectAll("path").each(function() { twoSlicePathStrings.push(d3.select(this).attr("d")); });
-      assert.lengthOf(twoSlicePathStrings, twoSliceData.length * 2, "2 paths per datum");
-
-      twoSlicePathStrings.forEach((pathString, index) => {
-        assert.notStrictEqual(pathString, fourSlicePathStrings[index], "slices were updated when data changed");
+        let secondPathPoints1 = pathPoints1[2].split(",");
+        assert.closeTo(parseFloat(secondPathPoints1[0]), 0, 1, "draws line to origin");
+        assert.closeTo(parseFloat(secondPathPoints1[1]), 0, 1, "draws line to origin");
+        svg.remove();
+      });
+
+      it("uses a plottable-colors color scale to color sectors by default", () => {
+        let arcPaths = piePlot.content().selectAll(".arc");
+
+        let arcPath0 = d3.select(arcPaths[0][0]);
+        assert.strictEqual(arcPath0.attr("fill"), "#5279c7", "first sector filled appropriately");
+
+        let arcPath1 = d3.select(arcPaths[0][1]);
+        assert.strictEqual(arcPath1.attr("fill"), "#fd373e", "second sector filled appropriately");
+        svg.remove();
+      });
+
+      it("updates slices when data changes", () => {
+        let originalPathStrings: String[] = [];
+        piePlot.content().selectAll("path").each(function() { originalPathStrings.push(d3.select(this).attr("d")); });
+        assert.lengthOf(originalPathStrings, simpleData.length * 2, "2 paths per datum");
+
+        let oneMoreSliceData = simpleData.slice();
+        oneMoreSliceData.push({ value: 5 });
```

Comment:
collapse in the above. That way the name of the variable makes sense from first reading.